lib/src/screens/new_wallet/new_wallet_page.dart

```diff
@@ -92,44 +98,76 @@ class _WalletNameFormState extends State<WalletNameForm> {
               padding: EdgeInsets.only(left: 12, right: 12),
               child: AspectRatio(
                   aspectRatio: aspectRatioImage,
-                  child: FittedBox(child: widget.walletImage, fit: BoxFit.fill)),
+                  child:
+                      FittedBox(child: widget.walletImage, fit: BoxFit.fill)),
             ),
             Padding(
               padding: EdgeInsets.only(top: 24),
               child: Form(
-                  key: _formKey,
-                  child: TextFormField(
-                      onChanged: (value) => _walletNewVM.name = value,
-                      textAlign: TextAlign.center,
-                      style: TextStyle(
-                          fontSize: 20.0,
-                          fontWeight: FontWeight.w600,
-                          color:
-                              Theme.of(context).primaryTextTheme.title.color),
-                      decoration: InputDecoration(
-                          hintStyle: TextStyle(
-                              fontSize: 18.0,
-                              fontWeight: FontWeight.w500,
-                              color: Theme.of(context)
-                                  .accentTextTheme
-                                  .display3
-                                  .color),
-                          hintText: S.of(context).wallet_name,
-                          focusedBorder: UnderlineInputBorder(
-                              borderSide: BorderSide(
-                                  color: Theme.of(context)
-                                      .accentTextTheme
-                                      .display3
-                                      .decorationColor,
-                                  width: 1.0)),
-                          enabledBorder: UnderlineInputBorder(
-                              borderSide: BorderSide(
-                                  color: Theme.of(context)
-                                      .accentTextTheme
-                                      .display3
-                                      .decorationColor,
-                                  width: 1.0))),
-                      validator: WalletNameValidator())),
+                key: _formKey,
+                child: TextFormField(
+                  onChanged: (value) => _walletNewVM.name = value,
+                  controller: _controller,
+                  textAlign: TextAlign.center,
+                  style: TextStyle(
+                      fontSize: 20.0,
+                      fontWeight: FontWeight.w600,
+                      color: Theme.of(context).primaryTextTheme.title.color),
+                  decoration: InputDecoration(
+                    suffixIcon: IconButton(
+                      onPressed: () async {
+                        final String rName = await generateName();
```

Comment:
Do not define type for `final` variables

Reply:
@mkyq removed the type